### DISCUSSION TITLE
Replication gets stuck

### DISCUSSION THREAD

**Discussion by ronag** [apache/couchdb#3745] [Q&A] [answered]
What can cause replication to get stuck?

I have 3 couch instances:

couchdb: couch 1.7
couchdb1: couch 3.1.1
couchdb2: couch 3.1.1

The replicators run in the couch 3.1.1 instances (see screenshots)

![Screen Shot 2021-09-10 at 10 49 45](https://user-images.githubusercontent.com/3065230/132827648-6d9e4c04-cc03-47c2-9cdc-02cab48f9cde.png)

![Screen Shot 2021-09-10 at 10 49 41](https://user-images.githubusercontent.com/3065230/132827654-c6c1762e-bca9-4385-b615-58bd8d76b4ab.png)

I've tried restarting the couch instances several times in the past few days but can't seem to get it to make progress.

**Comment by nickva** [comment, marked as answer]
See if there are any errors in the logs or in the _scheduler/jobs for those replications

If replications use a filter and it's fairly restrictive, there could be a large number of pending changes and very few writes.

Another reason could be a validate doc update (VDU) function which is restricting writes.

Without a more details job history and/or logs it's a bit hard to tell why state looks like that

This might also be better converted to a discussion for now
